zendframework/zend-cache/ZF2015-07.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Filesystem Permissions Issues in Multiple Components
+link:      http://framework.zend.com/security/advisory/ZF2015-07
+cve:       CVE-2015-5723
+branches:
+    2.4.x:
+        time:     2015-09-15 16:24:04
+        versions: [>=2.4.0,<2.4.8]
```

Comment:
Older versions are affected; however, I'm uncertain how to indicate those.
Do I need a "branch" entry per minor version? How do I indicate all
versions on a minor branch?
On Sep 16, 2015 3:31 AM, "Christophe Coevoet" <notifications@github.com>
wrote:

> In zendframework/zend-cache/ZF2015-07.yaml
> <https://github.com/FriendsOfPHP/security-advisories/pull/104#discussion_r39604923>
> :
>
> > @@ -0,0 +1,11 @@
> > +title:     Filesystem Permissions Issues in Multiple Components
> > +link:      http://framework.zend.com/security/advisory/ZF2015-07
> > +cve:       CVE-2015-5723
> > +branches:
> > +    2.4.x:
> > +        time:     2015-09-15 16:24:04
> > +        versions: [>=2.4.0,<2.4.8]
>
> Are older 2.x versions not affected ? If they are, the lower bound should
> be changed, even if there is no fixed version in the 2.3 serie
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FriendsOfPHP/security-advisories/pull/104/files#r39604923>
> .
>